src/google/protobuf/compiler/main.cc

```diff
@@ -101,6 +105,29 @@ int ProtobufMain(int argc, char* argv[]) {
 }  // namespace protobuf
 }  // namespace google
 
-int main(int argc, char* argv[]) {
+#ifdef _MSC_VER
+std::string ToMultiByteUTF8String(const wchar_t* input)
```

Comment:
google3 naming convention is `ToMultiByteUtf8String` (note the lowercase on `tf`)

Reply:
fixed in [28dbcf5](https://github.com/protocolbuffers/protobuf/pull/14253/commits/28dbcf5b7e30243a296f795a5a7c16f51a99f6b8)